pool/cache.go

```diff
@@ -8,6 +8,10 @@ import (
 	"github.com/nspcc-dev/neofs-sdk-go/session"
 )
 
+var (
+	cacheSize = 700
```

Comment:
why do we have it here? mb add some comments? add "default" word to that var name?

Reply:
This value was hardcoded before in the constructor. In fact, this is a default and only one possible value, we don't pass/overwrite it from outside. 

- Is it a problem? Possible. 
- Should we change it and pass the value from outside? May be.
- Should we do these changes in this PR? No.

I left comments about these changes in the commit message. I'm not completely sure about the new size, if we have reasons to change it from 700 to another value, suggest me, please.

Reply:
> Should we do these changes in this PR? No.

of course not. just want to highlight that moving did not help much: it _was_ a magic number and it _is_ a magic number now. consider `sessionCacheSize`/`defaultSessionCacheSize`. we use caches often, and just `cacheSize` does not say anything about the cache's items

Reply:
This argument make sense, renamed to `defaultSessionCacheSize`

Reply:
#472.